### DISCUSSION TITLE
能否增加OpenAI的API让AI机器人可以更智能的聊天

### DISCUSSION THREAD

**Discussion by oiuv** [mod-playerbots/mod-playerbots#188] [Ideas]
现在把机器人加入公会后，聊天时他们会发空内容，能否增加配置，当配置了开启chatGPT之类的密钥和代理后，可以让机器人调用大模型实现更智能的交流互动，这个应该可以实现吧。

**Comment by Dreathean** [comment]
See progress on https://github.com/DustinHendrickson/mod-ollama-chat
